docs/docs/contributing/getting-started.md

```diff
@@ -83,9 +83,22 @@ Generating Documentation
 -------------------------
 To generate this page and other static page docs, run
 ```bash
+# the generated docs will be located in scaladoc/output
 $ sbt
-> genDocs
+> scalaDoc/generateScalaDocumentation
```

Comment:
This should be `scaladoc` instead of `scalaDoc`, no?

Reply:
yes, it is scaladoc